doc/usage_rsmeval.rst

```diff
@@ -55,7 +55,7 @@ This folder contains the final RSMEval report in HTML format as well as in the f
 
 output
 ~~~~~~
-This folder contains all of the :ref:`intermediate files <intermediate_files_rsmeval>` produced as part of the various analyses performed, saved as ``.csv`` files.
+This folder contains all of the :ref:`intermediate files <intermediate_files_rsmeval>` produced as part of the various analyses performed, saved as ``.csv`` files. ``rsmeval`` will also save in this folder a copy of the :ref:`configuration file <config_file_rsmeval>`. Fields not specified in the original configuration file will be pre-populated with default values. 
 
```

Comment:
I think what we want is to save the fields with default values relevant to the tool but skip the irrelevant fields. For example, for `rsmeval` you would have `'use_scaled_scores': False` if you did not specify this field but you will not have `train_label_column` field which is not relevant to this tool.